### DISCUSSION TITLE
Add active label classname to the tabs component

### DISCUSSION THREAD

**Discussion by kaaancan** [mantinedev/mantine#3396] [Feature requests]
For the SegmentedControl component there is a classNames property 'labelActive' which makes it easier to have custom css styling for the elements that are active. 

Something similar for the tabs component would be awesome.

I know that there is a simple workaround with just keeping the current tabs state and checking that or using the data-active attribute . But having that classname would still be very nice.

**Comment by rtivital** [comment]
Classes that represent component state will not be added. Actually, they will be removed from all components in 6.0 and replaced with data-* attributes. Using data attributes avoids issues with specificity (when default styles may override state styles).
